spec/namespaces/snapshot.yaml

```diff
@@ -359,6 +359,12 @@ components:
                 type: string
               rename_replacement:
                 type: string
+              rename_alias_pattern:
```

Comment:
Take the time to add a `description:` field to these. Eventually they will appear in the documentation.

Reply:
I synced this with the current documentation description and parameters.